lib/inflecto.rb

```diff
@@ -272,7 +272,7 @@ def self.tableize(input)
   #   Inflecto.classify("posts")        # => "Post"
   #
   #   # Singular names are not handled correctly:
-  #   Inflecto.classify("business")     # => "Busines"
+  #   Inflecto.classify("business")     # => "Business"
```

Comment:
does the code actually do this?

Reply:
I've seen you opened other PRs to other repos, so your activity could be based by spell check bot. This instance AFAIK actually documents a misspelling generated by the code.

Reply:
Sorry about that.

Reply:
np, had a positive intend I'm sure.